po/fr.po

```diff
@@ -104,7 +117,7 @@ msgstr "Cycler sur les fenêtres + réduire"
 
 #: Settings.ui.h:9
 msgid "Behavior for Middle-Click."
-msgstr "Comportement du clic molette"
+msgstr "Comportement du clic molette."
```

Comment:
Why did you add a dot there?

Reply:
It is good practice to follow the original message's structure. There's a dot in the original message, therefore there is a dot in the translated message.

Furthermore, POEdit complains if I don't put it there...

Reply:
Sure :)